NEWS.md

```diff
@@ -4,6 +4,10 @@
 
 * Add custom SQLite translation of `full_join()` and `right_join()` (#536, @mgirlich).
 
+* `select()` now does not relocate grouping variables to the front (#568, @mgirlich).
```

Comment:
Sorry, I didn't spot this earlier, but our convention is name first, then issue number.